api/mock_trace_service.proto

```diff
@@ -11,6 +11,14 @@ message GetNumSpansResponse {
     int32 num_spans = 1;
 }
 
+// Next available id: 2
+message SetDelayRequest {
+    // The duration of the delay in milliseconds.
+    int32 duration = 1;
```

Comment:
Prefer a defined duration type. It's easy for people to accidentally set this to seconds.
https://developers.google.com/protocol-buffers/docs/reference/csharp/class/google/protobuf/well-known-types/duration

If you _have_ to use a integer then it should be called durationMillis so that mistakes are more obvious:
```
// looks okay:
request.duration = delay.getSeconds()

// clearly wrong:
request.durationMillis = delay.getSeconds()
```

Reply:
Got it, switched to using `google.protobuf.Duration`